ui/tests/unit/serializers/kv/metadata-test.js

```diff
@@ -14,28 +14,55 @@ module('Unit | Serializer | kv/metadata', function (hooks) {
     const serverData = {
       request_id: 'foo',
       backend: 'my-backend',
+      path: '',
       data: {
-        keys: ['first', 'second'],
+        keys: ['first', 'second', 'third/'],
       },
     };
     const expectedData = [
       {
         id: 'my-backend/metadata/first',
         path: 'first',
-        backend: 'my-backend',
+        full_secret_path: 'first',
```

Comment:
We don't need it on the individual secret models for the metadata list view. Because the route is already aware of the backend ex: kv-engine. We may need it later down the line on other metadata calls, but I can't really visualize how'd we'd need it, so until we do I'm taking it off. Then if we do need it we can add it back. Thoughts?

Reply:
Personally I think since have access to it we should have it on the model, especially since we already did. But i don't feel strongly

Reply:
Good point about it being on the model previously. I'll add it back.